examples/browser/package.json

```diff
@@ -13,18 +13,26 @@
     "theia": "file:../../../Theia"
   },
   "scripts": {
-    "clean": "./node_modules/.bin/rimraf lib",
-    "lint": "./node_modules/.bin/tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
+    "clean": "rimraf lib",
```

Comment:
Indeed, the `./node_modules/.bin` directory is added in front of the path, so it will always use the local one first.

A reason for using `./node_modules/.bin/foo` over just `foo` is that if I have `foo` installed globally, I may forget to add it to dependencies/devDependencies and not notice it.